test/functional/android/ime_tests.py

```diff
@@ -45,7 +43,11 @@ def test_is_ime_active(self):
         self.assertTrue(self.driver.is_ime_active())
 
     def test_active_ime_engine(self):
-        self.assertIsInstance(self.driver.active_ime_engine, unicode)
+        try:
```

Comment:
You also can modify like:

```
engines = self.driver.available_ime_engines
self.assertTrue(self.driver.active_ime_engine in engines)
```

instead of adding except.

Reply:
Though I tried to keep current verification, your idea is better.

* (Before) Check if type is expected(```basestring```, ```str```)
* (After) Check if the value is within ```available_ime_engines```.